lib/improver/wxcode/weather_symbols.py

```diff
@@ -395,6 +434,10 @@ def find_all_routes(graph, start, end, route=None):
                 heavy_precipitation).
             end (int):
                 The weather symbol code to which we are tracing all routes.
+            missing_nodes (dict) or None:
```

Comment:
I think this is semantics. I think of missing_nodes as nodes I want to miss out because the data is missing. Happy to change this to omit_nodes if you think that is clearer.

Reply:
Yes please. I think missing_nodes is likely to be misleading, and as I'm probably going to be working with this code, I'd like to make my life easier. Thanks